src/communication/network_interface_cl.c

```diff
@@ -1840,7 +1840,7 @@ file_apply_tde_to_class_files (const OID * class_oid)
 #endif /* !CS_MODE */
 }
 
-#ifdef UNSTABLE_TDE_FOR_REPLICATION_LOG
+#if defined(UNSTABLE_TDE_FOR_REPLICATION_LOG) || defined(CS_MODE)
```

Comment:
If `defined(CS_MODE)` should be added, we can remove this line.

Reply:
Do you mean "#if defined(UNSTABLE_TDE_FOR_REPLICATION_LOG) || defined(CS_MODE)"  same as  "#if 1" ?

Reply:
Is this a function that hasn't been used before? 
Why is UNSTABLE_TDE_FOR_REPLICATION_LOG written?

Reply:
@shparkcubrid 
UNSTABLE_TDE_FOR_REPLICATION_LOG is a declaration to keep the code developed by the 2nd team.
It has not been completed, and so far, there are no plans to activate it in the future.